apps/src/applab/applab.js

```diff
@@ -2905,6 +2946,11 @@ Applab.handleReadRecords = function(successCallback, records) {
 };
 
 Applab.updateRecord = function (opts) {
+  apiValidateType(opts, 'updateRecord', 'table', opts.table, 'string');
+  apiValidateType(opts, 'updateRecord', 'record', opts.record, 'object');
+  apiValidateTypeAndRange(opts, 'updateRecord', 'record.id', opts.record.id, 'number', 1, Infinity);
```

Comment:
Do you want to validate that the number is an integer as well? Perhaps a new "integer" type that behaves like "number" but a little different?